PerimeterXModule/Internals/Helpers/PxCommonUtils.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Net;
+using System.Web;
+
+namespace PerimeterX
+{
+	class PxCommonUtils
+	{
+		/**
+		 * <summary>
+		 * Request helper, extracting the ip from the request according to sockerIpHeader or from 
+		 * the request socker when sockerIpHeader is absent
+		 * </summary>
+		 * <param name="context" >HttpContext</param>
+		 * <param name="pxConfig">PxConfiguration</param>
+		 * <returns>Ip from the request</returns>
```

Comment:
Nitpicking - here in the comment you can feel safe to call it IP (instead of Ip or ip) :) (also relevant to the `<summary>` part above).

Reply:
👎